docs/api/client/http-api.md

```diff
@@ -6,6 +6,65 @@ description: ReSolve provides a standard HTTP API that allows you to send aggreg
 
 ReSolve provides a standard HTTP API that allows you to send aggregate commands, and query Read and View Models.
```

Comment:
Please rewrite without using "provides".

Reply:
Same as above, changed to "ReSolve exposes an HTTP API that allows you to send aggregate commands, and query Read and View Models."